src/Proxy.hs

```diff
@@ -1135,3 +1295,39 @@ getHealthR =
 
 getIpsR :: Handler TypedContent
 getIpsR = toTypedContent . ipInfo <$> getYesod
+
+
+getBakerPoolR :: Word64 -> Handler TypedContent
+getBakerPoolR bid =
+    runGRPC doGetBaker $ \v -> do
+      $(logInfo) "Successfully got baker pool status."
+      sendResponse v
+  where
+    doGetBaker = withLastFinalBlockHash Nothing (getPoolStatus (BakerId $ AccountIndex bid) False)
+
+getChainParametersR :: Handler TypedContent
+getChainParametersR =
+    runGRPC doGetParameters $ \(v :: Parameters.ChainParameters' 'ChainParametersV1) -> do
+      $(logInfo) "Successfully got chain parameters."
+      sendResponse $ toJSON v
```

Comment:
It would be useful to have a better error message here in case the node is running the old protocol version at the moment.